lib/posix/posix_linux_amd64_consts.nim

```diff
@@ -634,7 +637,7 @@ const SC_MQ_OPEN_MAX* = cint(27)
 const SC_MQ_PRIO_MAX* = cint(28)
 const SC_NGROUPS_MAX* = cint(3)
 const SC_OPEN_MAX* = cint(4)
-const SC_PAGE_SIZE* = cint(30)
+const SC_PAGESIZE* = cint(30)
```

Comment:
Why the rename? Doesn't this break backwards compatibility?

Reply:
theoretically it shouldn't because of nim's identifier resolution. [both are valid apparently?](http://man7.org/linux/man-pages/man3/sysconf.3.html)

```
PAGESIZE - _SC_PAGESIZE
              Size of a page in bytes.  Must not be less than 1.

PAGE_SIZE - _SC_PAGE_SIZE
              A synonym for PAGESIZE/_SC_PAGESIZE.  (Both PAGESIZE and
              PAGE_SIZE are specified in POSIX.)
```